### PR TITLE
new add Arrays

### PR DESCRIPTION
**Pull request by viraj19r**
add new folder array

### CODE REVIEW

File: Maximilian _udemy/array/foreach_and_map__method.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+//     taxAmount.push(elements*(1+tax));
+// }
+// amount.forEach((taxAm,index,wholeArray)=>{ // forEach(all values of array,index of array, whole array) [it points to the same memory so we cannot assign it to a new variable]
+//    taxAmount.push(taxAm*(1+tax));  
+// });
+console.log(taxAmount);
+const addTaxToAmount = amount.map((taxAm,index,wholeArray)=>{ // map will now return(it must return something) a new array pointing to a new memory. so we can store in a variable
+  return  taxAmount.push(taxAm*(1+tax));  
+ });
+ console.log(taxAmount);//
```
Comment (viraj19r) [inline Maximilian _udemy/array/foreach_and_map__method.js]: use prettier for all these files

File: Maximilian _udemy/array/foreach_and_map__method.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+const taxAmount = [];
+// for (const elements of amount){
+//     taxAmount.push(elements*(1+tax));
+// }
+// amount.forEach((taxAm,index,wholeArray)=>{ // forEach(all values of array,index of array, whole array) [it points to the same memory so we cannot assign it to a new variable]
+//    taxAmount.push(taxAm*(1+tax));  
+// });
+console.log(taxAmount);
+const addTaxToAmount = amount.map((taxAm,index,wholeArray)=>{ // map will now return(it must return something) a new array pointing to a new memory. so we can store in a variable
+  return  taxAmount.push(taxAm*(1+tax));
```
Comment (viraj19r) [inline Maximilian _udemy/array/foreach_and_map__method.js]: map itself returns a new array, remove use of push in map

File: Maximilian _udemy/array/sort_reverse_filter_method.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+const amount = [10.7, 4, 5.6, 3, 6, 2.66];
+amount.sort((a, b) => {
```
Comment (viraj19r) [inline Maximilian _udemy/array/sort_reverse_filter_method.js:2]: using ternary here would simplify this logic

**Review by viraj19r — COMMENTED** [review]
-

**Review by rvashishth — APPROVED** [review]

**Pull request closed without merging**